python/py-ligo-common/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem         1.0
+PortGroup          python 1.0
+
+name               py-ligo-common
+version            1.0.3
+
+categories-append  science
+platforms          darwin
+supported_archs    noarch
+maintainers        {ram @skymoo} {ligo.org:duncan.macleod @duncanmmacleod}
+license            GPL
+
+description        Common package for LIGO Python libraries
+long_description   ${description}
+homepage           https://git.ligo.org/lscsoft/ligo-common
+
+master_sites       http://software.ligo.org/lscsoft/source/
+distname           ligo-common-${version}
+
+checksums  rmd160  ba7b27ac876e1133e852fd9f3f8bb2442bb24638 \
+           sha256  a3e00d79bf3b0474b429f50fb60079da015453afa09658f90efcab8ae158a835 \
+           size    15620
+
+python.versions         27 36
+python.default_version  27
```

Comment:
Just noting that the point where 2.7 goes off support is creeping up, so I recommend to people that 3.6 is a better thing to be aiming at these days. You don't _need_ to do that though.

Reply:
@pmetzger, the LIGO experiment is still based primarily off python2.7, so I would like to keep that as the default until we can move to a base on python3.x.

That said, I don't really understand what impact this `default_version` setting has, presumably its just defining what happens when people try to install `py-ligo-common`?

Reply:
It's not a big deal. You don't have to alter this if you don't want to.

Reply:
For LIGO we are really starting to push on Python3 but at the moment we simply can't get enough traction to move over at this time. It is on the TODO list and something we are working on. Hence moving the `ligo-common` port over to provide a Python3 version.